lib/habits_web/models/habit.ex

```diff
@@ -6,8 +6,8 @@ defmodule HabitsWeb.Habit do
 
   use Habits.Web, :model
 
-  alias Habits.Repo
-  alias HabitsWeb.{Account, CheckIn, Streak}
+  alias Habits.{Repo, Accounts.Account}
```

Comment:
I do regret the need for `alias` here to reach deep into the `Accounts` context to get the schema module. I wonder if it would be reasonable to define a `Accounts.schema` function that returns the `Accounts.Account` module name...